plugins/source/azure/resources/services/sql/database_vulnerability_assesment_scans.go

```diff
@@ -0,0 +1,54 @@
+// Code generated by codegen; DO NOT EDIT.
```

Comment:
Let's remove this `// Code generated by codegen; DO NOT EDIT.` header (also others if exist - I didn't look for others)